### ISSUE TITLE
help with installing theme(macos big sur)

### ISSUE THREAD

**Issue by Betweenwonders** [React95/React95#314]
when I ran code npm install @react95/core @react95/gatsby-theme styled-components these error messages came up:

---

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: betweenwonders@1.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.14.0" from @react95/core@5.7.2
npm ERR! node_modules/@react95/core
npm ERR!   @react95/core@"^5.7.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/betweenwinters/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/betweenwinters/.npm/_logs/2022-01-22T10_38_34_364Z-debug.log
Freyas-MacBook-Pro:betweenwonders betweenwinters$  

----

Any help is extremely appreciated!

**Comment by ggdaltoso** [comment]
hey @Betweenwonders !

Can you share a repo to help in reproducing this behavior?

**Comment by andomain** [comment]
Looks like this NPM module doesn't currently support React 17+
Easy step to reproduce this is to bootstrap a React app using up to date CRA and attempt to install the package.

```
npx create-react-app reproduce-dependency-issue
cd reproduce-dependency-issue
npm i @react95/core
```

**Comment by ggdaltoso** [comment]
@andomain I think it would be addressed in #317 since we just need to bump react version in @react95/core.

@mavericardo is working on it but feel free to sync with he or even opening a PR just to fix this.

I'm glad to see this is a easy solution, btw!
